canonical-web-and-design/snapcraft.io
# Automate publisher pages

## QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/publisher/kde
- Visit http://0.0.0.0:8004/publisher/jetbrains
- KDE page should have more snaps compared to production
- Jetbrains page should be the same
